counsel.el

```diff
@@ -4047,6 +4044,7 @@ the current outline heading. See `counsel-outline-settings'."
 current outline heading in org-mode buffers. See
 `counsel-outline-settings'."
   (apply 'org-get-heading (counsel--org-get-heading-args)))
+  (apply #'org-get-heading (counsel--org-get-heading-args)))
```

Comment:
The same expression is called twice. Typo?

Reply:
Oops, thanks. It's an artefact of trying to split the doc and code changes into separate commits, which is fixed by the subsequent doc fix. I'll clean up the commits now.